packages/editor/src/components/editor-canvas/edit-template-blocks-notification.js

```diff
@@ -27,14 +27,31 @@ import { store as editorStore } from '../../store';
  *                                                                  editor iframe canvas.
  */
 export default function EditTemplateBlocksNotification( { contentRef } ) {
-	const renderingMode = useSelect(
-		( select ) => select( editorStore ).getRenderingMode(),
+	const { renderingMode, getPostLinkProps, templateId } = useSelect(
+		( select ) => {
+			const {
+				getRenderingMode,
+				getEditorSettings,
+				getCurrentTemplateId,
+			} = select( editorStore );
+			return {
+				renderingMode: getRenderingMode(),
+				getPostLinkProps: getEditorSettings().getPostLinkProps,
+				templateId: getCurrentTemplateId(),
+			};
+		},
 		[]
 	);
+	const editTemplate = getPostLinkProps
+		? getPostLinkProps( {
+				postId: templateId,
+				postType: 'wp_template',
+				canvas: 'edit',
```

Comment:
Why are we passing canvas edit here. The editor package shouldn't know about such thing.

Reply:
It was needed in order to navigate to the edit page rather than the summary view with left panel, but you are correct, this was not a good place to set this, have pulled it [back into the site editor package](https://github.com/WordPress/gutenberg/pull/57700/commits/a04ff70af18805a4e3d7a82df7f6213df3dea57a).